### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Anubhavdevv**
## What does this PR do?

Enhanced the documentation by wrapping hyperlink to the official docs link

## Related PRs and Issues

#131 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # The Appwrite Docs 📝
 
-The official https://appwrite.io/docs documentation source code.
+The official [Appwrite Docs](https://appwrite.io/docs) documentation source code.
```
Comment (gewenyu99) [inline README.md:3]:
```suggestion
The official [Appwrite Documentation](https://appwrite.io/docs) source code.
```

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Hey! 

Thanks for contributing to Appwrite.

I have a small suggestion because the current wording says documentation twice. Otherwise, a great idea :)

File: README.md (hunk 1 above)
Comment (Anubhavdevv) [reply README.md:3]: Sure Thanks for the suggestion !

**Comment by eldadfux** [comment]
@Anubhavdevv thanks for contributing to Appwrite!

**Comment by Anubhavdevv** [comment]
Thank you @eldadfux 🙌
Looking forward to contribute more to the Community !

**Pull request merged**